packages/trackerless-network/src/logic/proxy/ProxyClient.ts

```diff
@@ -58,19 +58,21 @@ interface ProxyDefinition {
 
 const logger = new Logger(module)
 
-export class ProxyStreamConnectionClient extends EventEmitter {
+const SERVICE_ID = 'system/proxy-client'
```

Comment:
Better to use `layer2-${config.streamPartId}`. The `layer2` prefix need s to be used here as the proxy clients use the same service as random graph nodes for message propagation. The prefix could be an exported constant such as `STREAM_PART_DELIVERY_SERVICE_ID_PREFIX`

Reply:
The value can also be changed `layer2` -> `stream-part-delivery`